<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QWidget" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>-10</y>
     <width>800</width>
     <height>600</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>22</pointsize>
     <stylestrategy>PreferAntialias</stylestrategy>
    </font>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="sivu1">
    <widget class="QToolButton" name="kirjaudu_ulos">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>450</y>
       <width>371</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>22</pointsize>
       <stylestrategy>PreferAntialias</stylestrategy>
      </font>
     </property>
     <property name="text">
      <string>Log out</string>
     </property>
    </widget>
    <widget class="QToolButton" name="tilitapahtumat">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>260</y>
       <width>351</width>
       <height>101</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>22</pointsize>
       <stylestrategy>PreferAntialias</stylestrategy>
      </font>
     </property>
     <property name="text">
      <string>Tilitapahtumat</string>
     </property>
    </widget>
    <widget class="QToolButton" name="talletus">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>70</y>
       <width>351</width>
       <height>101</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>22</pointsize>
       <stylestrategy>PreferAntialias</stylestrategy>
      </font>
     </property>
     <property name="text">
      <string>Talletus</string>
     </property>
    </widget>
    <widget class="QToolButton" name="nosto">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>351</width>
       <height>101</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>22</pointsize>
       <stylestrategy>PreferAntialias</stylestrategy>
      </font>
     </property>
     <property name="text">
      <string>Nosto</string>
     </property>
    </widget>
    <widget class="QPushButton" name="saldo_nappi">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>260</y>
       <width>351</width>
       <height>101</height>
      </rect>
     </property>
     <property name="text">
      <string>Saldo</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="sivu2">
    <widget class="QListView" name="listView">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>40</y>
       <width>491</width>
       <height>551</height>
      </rect>
     </property>
     <property name="gridSize">
      <size>
       <width>4</width>
       <height>10</height>
      </size>
     </property>
     <property name="itemAlignment">
      <set>Qt::AlignLeading</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="account_name">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>251</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="account_lable">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>40</y>
       <width>171</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
       <stylestrategy>PreferAntialias</stylestrategy>
      </font>
     </property>
     <property name="text">
      <string>Account Owner</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="balance">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>150</y>
       <width>251</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="balance_label_2">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>120</y>
       <width>181</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
       <stylestrategy>PreferAntialias</stylestrategy>
      </font>
     </property>
     <property name="text">
      <string>Account Balance</string>
     </property>
    </widget>
    <widget class="QPushButton" name="prev_10">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>410</y>
       <width>250</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>Prev 10 actions</string>
     </property>
    </widget>
    <widget class="QPushButton" name="next_10">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>460</y>
       <width>250</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>Next 10 actions</string>
     </property>
    </widget>
    <widget class="QPushButton" name="close">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>510</y>
       <width>250</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>close</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="sivu3">
    <widget class="QLineEdit" name="saldoArvo_2">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>70</y>
       <width>301</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="name">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>301</width>
       <height>51</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="name_label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>201</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Account Owner</string>
     </property>
    </widget>
    <widget class="QLabel" name="balance_label">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>30</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Balance</string>
     </property>
    </widget>
    <widget class="QListView" name="transaction">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>210</y>
       <width>751</width>
       <height>271</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="transaction_label">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>170</y>
       <width>251</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Last 5 transactions</string>
     </property>
    </widget>
    <widget class="QPushButton" name="close_button">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>520</y>
       <width>281</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Close</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="sivu4">
    <widget class="QToolButton" name="takaisin">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>510</y>
       <width>371</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>22</pointsize>
       <stylestrategy>PreferAntialias</stylestrategy>
      </font>
     </property>
     <property name="text">
      <string>takaisin</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="nostoArvo">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>340</y>
       <width>261</width>
       <height>81</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
       <stylestrategy>PreferAntialias</stylestrategy>
      </font>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="text">
      <string>0.0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="clear">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>430</y>
       <width>261</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>22</pointsize>
       <stylestrategy>PreferAntialias</stylestrategy>
      </font>
     </property>
     <property name="text">
      <string>clear</string>
     </property>
    </widget>
    <widget class="QToolButton" name="nostaNappi">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>510</y>
       <width>371</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>22</pointsize>
       <stylestrategy>PreferAntialias</stylestrategy>
      </font>
     </property>
     <property name="text">
      <string>nosta</string>
     </property>
    </widget>
    <widget class="QLabel" name="teksti_2">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>260</y>
       <width>161</width>
       <height>81</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>36</pointsize>
       <stylestrategy>PreferAntialias</stylestrategy>
      </font>
     </property>
     <property name="text">
      <string>Nosto</string>
     </property>
    </widget>
    <widget class="QWidget" name="gridLayoutWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>421</width>
       <height>461</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="spacing">
       <number>20</number>
      </property>
      <item row="0" column="1">
       <widget class="QPushButton" name="nosto1">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <pointsize>26</pointsize>
          <stylestrategy>PreferAntialias</stylestrategy>
         </font>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>40</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="nosto5">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <pointsize>26</pointsize>
          <stylestrategy>PreferAntialias</stylestrategy>
         </font>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>500</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="nosto2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <pointsize>26</pointsize>
          <stylestrategy>PreferAntialias</stylestrategy>
         </font>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>60</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="nosto3">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <pointsize>26</pointsize>
          <stylestrategy>PreferAntialias</stylestrategy>
         </font>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>100</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="nosto4">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <pointsize>26</pointsize>
          <stylestrategy>PreferAntialias</stylestrategy>
         </font>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>200</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QPushButton" name="nosto0">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <pointsize>26</pointsize>
          <stylestrategy>PreferAntialias</stylestrategy>
         </font>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>20</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QLineEdit" name="saldoArvo">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>80</y>
       <width>261</width>
       <height>81</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
       <stylestrategy>PreferAntialias</stylestrategy>
      </font>
     </property>
     <property name="text">
      <string>0.0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="saldo">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>10</y>
       <width>201</width>
       <height>71</height>
      </rect>
     </property>
     <property name="text">
      <string>Saldo</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="sivu5">
    <widget class="QToolButton" name="takaisin_talletus">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>480</y>
       <width>371</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>22</pointsize>
       <stylestrategy>PreferAntialias</stylestrategy>
      </font>
     </property>
     <property name="text">
      <string>takaisin</string>
     </property>
    </widget>
    <widget class="QPushButton" name="talleta_20">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>350</y>
       <width>91</width>
       <height>71</height>
      </rect>
     </property>
     <property name="text">
      <string>20</string>
     </property>
    </widget>
    <widget class="QPushButton" name="talleta_40">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>350</y>
       <width>91</width>
       <height>71</height>
      </rect>
     </property>
     <property name="text">
      <string>40</string>
     </property>
    </widget>
    <widget class="QPushButton" name="talleta_500">
     <property name="geometry">
      <rect>
       <x>680</x>
       <y>350</y>
       <width>91</width>
       <height>71</height>
      </rect>
     </property>
     <property name="text">
      <string>500</string>
     </property>
    </widget>
    <widget class="QPushButton" name="talleta_200">
     <property name="geometry">
      <rect>
       <x>560</x>
       <y>350</y>
       <width>91</width>
       <height>71</height>
      </rect>
     </property>
     <property name="text">
      <string>200</string>
     </property>
    </widget>
    <widget class="QPushButton" name="talleta_60">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>350</y>
       <width>91</width>
       <height>71</height>
      </rect>
     </property>
     <property name="text">
      <string>60</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="talletusArvo">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>190</y>
       <width>401</width>
       <height>91</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
       <stylestrategy>PreferAntialias</stylestrategy>
      </font>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="text">
      <string>0.0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="talleta_100">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>350</y>
       <width>91</width>
       <height>71</height>
      </rect>
     </property>
     <property name="text">
      <string>100</string>
     </property>
    </widget>
    <widget class="QPushButton" name="clear2">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>190</y>
       <width>101</width>
       <height>91</height>
      </rect>
     </property>
     <property name="text">
      <string>clear</string>
     </property>
    </widget>
    <widget class="QLabel" name="teksti">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>30</y>
       <width>171</width>
       <height>81</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>36</pointsize>
       <stylestrategy>PreferAntialias</stylestrategy>
      </font>
     </property>
     <property name="text">
      <string>Talletus</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
